### PR TITLE
[ADD] pos_mercado_pago: integration of Mercado Pago payment terminal

### PR DESCRIPTION
**Pull request by papyDoctor**
Doc Point Smart

mool

### CODE REVIEW

**Comment by papyDoctor** [comment]
@LoredanaLrpz

**Comment by LoredanaLrpz** [comment]
--> saas-17.3: enable in app settings and not module

File: content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+#. Set your Point Smart terminal to the :guilabel:`Point of Sale` operation mode.
+
+   .. warning::
+      Odoo does not support the Point Smart :guilabel:`Standalone` operation mode.
+#. :ref:`Generate your credentials <mercado_pago/credentials>`
+#. :ref:`Create and configure the related payment method <mercado_pago/method>`
+
+.. _mercado_pago/credentials:
+
+Generate your credentials
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst:42]:
```suggestion
Generate credentials
```
[No pronouns in headings](https://www.odoo.com/documentation/17.0/contributing/documentation/content_guidelines.html#titles-and-headings) :)

File: content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
+============
+Mercado Pago
+============
+
+Connecting a payment terminal allows you to offer a fluid payment flow to your customers and ease
+the work of your cashiers.
+
+.. important::
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst:8]: Not sure all these bullet points are necessary, the _important_ markup takes a lot of space.

File: content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+Configuration
+=============
+
+#. Create your `Mercado Pago account <https://www.mercadopago.com.mx/>`_.
+#. Associate your terminal with a :guilabel:`store` and a :guilabel:`cash drawer` by following
+   `Mercado Pago's documentation <https://www.mercadopago.com.mx/developers/en/docs>`_.
+
+   .. note::
+      All the purchased terminals are automatically available on your Mercado dashboard.
+#. Set your Point Smart terminal to the :guilabel:`Point of Sale` operation mode.
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst:33]:
```suggestion

#. Set your Point Smart terminal to the :guilabel:`Point of Sale` operation mode.
```
While it doesn't seem to cause any issues, we usually separate markups with empty lines

File: content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+   - Fill in the :guilabel:`Production secret key` field using the webhook secret key.
+   - Fill in the :guilabel:`Terminal S/N` field using the terminal serial number. You can find it at
+     the back of your terminal.
+
+.. image:: mercado_pago/payment-method.png
+   :alt: form to create a new payment method
+
+You can select the payment method in your POS settings once the payment method is created. To do so,
+go to the :ref:`POS' settings <configuration/settings>` and add the payment method under the
+:guilabel:`Payment Methods` field of the :guilabel:`Payment` section.
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst:82]:
```suggestion
Select the payment method by
going to the :ref:`POS' settings <configuration/settings>` and adding it under the
:guilabel:`Payment Methods` field of the :guilabel:`Payment` section.
```
Suggestion: could be shortened

File: content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+Pay with a payment terminal
+===========================
+
+When processing a payment, select your Mercado Pago payment method, check the amount and click
+:guilabel:`Send`. Once the payment is successful, the status changes to :guilabel:`Payment
+Successful`.
+
+.. note::
+   - | In case of connection issues between Odoo and the payment terminal, force the payment by
+       clicking on :guilabel:`Force Done`, which allows you to validate the order.
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst:93]:
```suggestion
       clicking :guilabel:`Force Done`, which allows you to validate the order.
```
To be consistent (click or click on throughout)

File: content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+:guilabel:`Payment Methods` field of the :guilabel:`Payment` section.
+
+Pay with a payment terminal
+===========================
+
+When processing a payment, select your Mercado Pago payment method, check the amount and click
+:guilabel:`Send`. Once the payment is successful, the status changes to :guilabel:`Payment
+Successful`.
+
+.. note::
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst:91]: The bullet points are not necessary here, and the paragraphs add too much spacing.

**Review by xpl-odoo — APPROVED** [review]
Just some minor comments :)

**Comment by xpl-odoo** [comment]
@robodoo delegate=LoredanaLrpz

File: content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+Connecting a payment terminal allows you to offer a fluid payment flow to your customers and ease
+the work of your cashiers.
+
+.. important::
+   - Odoo only supports **Point Smart** payment terminals. They can be purchased on
+     `Mercado Pago's website <https://www.mercadopago.com.mx/herramientas-para-vender/lectores-point>`_.
+   - Point Smart can only be used in the following :abbr:`LATAM (Latin America)` countries:
+
+     - Argentina
```
Comment (ren-odoo) [inline content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst:13]: @LoredanaLrpz the scope is actully just Mexico, Argentina and Brazil, can you update it please?

**Comment by ren-odoo** [comment]
Hi @xpl-odoo do you know if there is someone that ca retake the open items on this PR?

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**